configure.ac

```diff
@@ -671,7 +676,7 @@ AC_CHECK_HEADERS([netinet/if_ether.h], [], [],
   ]]
 )
 
-AC_CHECK_HEADERS([net/pfvar.h],
+AC_CHECK_HEADERS([net/pfvar.h net/pf/pfvar.h],
```

Comment:
It will set have_net_pfvar_h="no" if 'net/pfvar.h' exists, and 'net/pf/pfvar.h' doesn't exist. Probably break on first find could improve it.
From autoconf documentation for AC_CHECK_HEADERS:
"If action-if-found is given, it is additional shell code to execute when one of the header files is found. You can give it a value of ‘break’ to break out of the loop on the first match."